_data/events.yml

```diff
@@ -31,6 +207,75 @@
 
 
 
+- name: "Performance oriented Spring Data JPA & Hibernate"
+  subTitle: ""
+  date: 2025-01-30
+  hide: true
+  track: "JavaTrack"
+  summary: >
+
+  level: Introduction
+  content: "Hands-On"
+  type: ["On-Site"]
+  duration: "4h"
+  video: ""
+  image: ""
+  gallery:
+    - ""
+  linkedIn: ""
+  instagram: ""
+  twitter: ""
+  githubUrl: ""
+  blogPost: ""
+
+
+
+- name: ".NET Sessie 1"
+  subTitle: ""
+  date: 2025-01-23
+  hide: true
+  track: "DotNetTrack"
+  summary: >
+
+  level: Introduction
+  content: "Hands-On"
+  type: ["On-Site"]
+  duration: "4h"
+  video: ""
+  image: ""
+  gallery:
+    - ""
+  linkedIn: ""
+  instagram: ""
+  twitter: ""
+  githubUrl: ""
+  blogPost: ""
+
+
+
+- name: "QA minded: support & sharesession"
+  subTitle: ""
+  date: 2025-01-09
+  hide: true
+  track: "QAEvent
+  summary: >
+
+  level: Introduction
+  content: "Hands-On"
+  type: ["On-Site"]
+  duration: "4h"
+  video: ""
+  image: ""
+  gallery:
+    - ""
+  linkedIn: ""
+  instagram: ""
+  twitter: ""
+  githubUrl: ""
+  blogPost: ""
+
+
```

Comment:
Also add a FunEvent here:
20/12/2024
Kerstdrink itenium
-- there should be many pictures for this one